src/services/syncFeedPerUser.js

```diff
@@ -197,23 +209,32 @@ const syncFeedPerUserProcess = async (userId) => {
   await syncMainFeedBroad(userId);
 };
 
-const followManyUserFeed = async (userId, userIdsToFollow, originFeed, targetFeed) => {
-  if (!userIdsToFollow || userIdsToFollow.length == 0) {
-    return;
+const syncMainFeedBroadPerUser = async (req, res) => {
+  try {
+    const {userId} = req.body;
+    await syncMainFeedBroad(userId);
+
+    return successResponse(res, 'sync data sucesfully', []);
+  } catch (error) {
+    return errorResponse(res, error.toString(), 500);
   }
-  const cs = stream.connect(process.env.API_KEY, process.env.SECRET);
+};
 
-  const payload = userIdsToFollow.map((ui) => ({
-    source: `${originFeed}:${userId}`,
-    target: `${targetFeed}:${ui}`
-  }));
+const syncFeedPerUser = async (req, res) => {
+  try {
+    const {userId} = req.body;
+    await syncFeedPerUserProcess(userId);
 
-  return await cs.followMany(payload);
+    return successResponse(res, 'sync data sucesfully', []);
+  } catch (error) {
+    return errorResponse(res, error.toString(), 500);
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, in the `syncFeedPerUser` function, there's no validation of the `userId` received from the request body. It's important to validate input data before processing it.
```diff
  const syncFeedPerUser = async (req, res) => {
    try {
      const {userId} = req.body;
+     if (!userId) {
+       return errorResponse(res, 'User ID is required', 400);
+     }
      await syncFeedPerUserProcess(userId);
  
      return successResponse(res, 'sync data sucesfully', []);
    } catch (error) {
      return errorResponse(res, error.toString(), 500);
    }
  };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->